code/modules/station_goals/bluespace_tap_lootpools.dm

```diff
@@ -0,0 +1,287 @@
+		/obj/item/gun/projectile/shotgun/toy/tommygun,
+		/obj/machinery/snow_machine,
+		/obj/item/clothing/head/kitty,
+		/obj/item/coin/antagtoken,
+		/obj/item/bedsheet/patriot,
+		/obj/item/bedsheet/rainbow,
+		/obj/item/bedsheet/captain,
+		/obj/item/bikehorn/golden,
+		/obj/item/salvage/ruin/brick,
+		/obj/item/toy/sword,
+		/obj/item/toy/windup_toolbox,
+		/obj/item/toy/foamblade,
+		/obj/item/toy/redbutton,
+		/obj/item/toy/figure/crew/ce,
+		/obj/item/toy/figure/crew/syndie,
+		/obj/item/stack/sheet/mineral/tranquillite/thirty,
+		/obj/item/stack/sheet/mineral/bananium/thirty
+	)
+
+/obj/effect/spawner/lootdrop/bluespace_tap/cultural_uncommon
+	name = "uncommon cultural artifacts"
+	loot = list(
+		/obj/item/gun/projectile/automatic/c20r/toy,
+		/obj/item/dualsaber/toy,
+		/obj/item/gun/projectile/automatic/toy/pistol/enforcer,
+		/obj/item/bedsheet/syndie,
+		/obj/item/bedsheet/wiz,
+		/obj/item/clothing/gloves/combat,
+		/obj/item/blank_tarot_card,
+		/obj/item/storage/box/syndie_kit/space,
+		/obj/item/wrench/brass,
+		/obj/item/crowbar/brass,
+		/obj/item/screwdriver/brass,
+		/obj/item/weldingtool/experimental/brass,
+		/obj/item/wirecutters/brass
+	)
+
+/obj/effect/spawner/lootdrop/bluespace_tap/cultural_rare
+	name = "rare cultural artifacts"
+	loot = list(
+		/obj/vehicle/space/speedbike/red,
+		/obj/item/gun/projectile/automatic/l6_saw/toy,
+		/obj/item/gun/projectile/automatic/sniper_rifle/toy,
+		/obj/item/bedsheet/centcom,
+		/obj/item/tarot_card_pack,
+		/obj/item/clothing/mask/holo_cigar,
+		/obj/item/melee/baseball_bat/homerun,
+		/obj/item/mod/module/dispenser,
+		/obj/item/storage/box/enforcer_rubber,
+		/obj/item/storage/belt/utility/brass
+	)
+
+/obj/effect/spawner/lootdrop/bluespace_tap/organic_common
+	name = "common organic objects"
+	loot = list(
+		/obj/item/soap/syndie,
+		/obj/item/dnainjector/telemut,
+		/obj/item/dnainjector/small_size,
+		/obj/item/dnainjector/morph,
+		/obj/item/dnainjector/regenerate,
+		/obj/item/dnainjector/firemut,
+		/obj/item/dnainjector/nobreath,
+		/obj/item/dnainjector/remoteview,
+		/obj/item/dnainjector/insulation,
+		/obj/item/dnainjector/eatmut,
+		/obj/item/dnainjector/noprints,
+		/obj/item/dnainjector/comic,
+		/obj/item/seeds/random/labelled,
+		/obj/item/organ/internal/regenerative_core,
+		/obj/item/reagent_containers/glass/bottle/reagent/omnizine,
+		/obj/item/reagent_containers/glass/bottle/reagent/lazarus_reagent
+	)
+
+/obj/effect/spawner/lootdrop/bluespace_tap/organic_uncommon
+	name = "uncommon organic objects"
+	loot = list(
+		/obj/item/storage/pill_bottle/random_meds/labelled,
+		/obj/item/dnainjector/hulkmut,
+		/obj/item/organ/internal/alien/plasmavessel,
+		/obj/item/organ/internal/alien/acidgland,
+		/obj/item/organ/internal/alien/hivenode,
+		/obj/item/organ/internal/alien/neurotoxin,
+		/obj/item/organ/internal/alien/resinspinner,
+		/obj/item/organ/internal/alien/eggsac,
+		/obj/item/organ/internal/heart/gland/heals,
```

Comment:
i was looking at the `heals` part confused why shoes were in the organic pool 💀

Reply:
Nope, just a healing organ!